src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -116,6 +110,11 @@ public function create(ContainerBuilder $container, string $id, array $config, ?
         $listener->replaceArgument(1, new Reference($rememberMeServicesId));
         $listener->replaceArgument(5, $config['catch_exceptions']);
 
+        // remember-me logout listener
+        $container->register(RememberMeLogoutListener::class)
```

Comment:
In core, we must always give an id to service - we don't do FQCN service (if we did, we should fix it). This allows proper separation of concerns (DI referencing vs class naming)

Why doesn't the id vary by id btw? If it's normal,can't the service be declared in XML?